test/e2e/websocket_test.go

```diff
@@ -122,7 +122,9 @@ func webSocketResponseFreqs(t *testing.T, clients *test.Clients, url string, num
 // (3) sends a message, and
 // (4) verifies that we receive back the same message.
 func TestWebSocket(t *testing.T) {
-	t.Parallel()
+	if !test.ServingFlags.HTTPS {
+		t.Parallel()
+	}
```

Comment:
Any reason why we don't run these tests in parallel?

Reply:
As it became flake if we webhook test with parallel, I added it. 
Let's see the result again without parallel.

Reply:
> As it became flake if we webhook test with parallel, I added it.

What do you mean?

Reply:
Here is the result when running tests with Parallel.
https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_serving/11322/pull-knative-serving-https/1395322718584311808
It fails to connect as:
```
    websocket.go:83: Connection failed: dial tcp 35.231.11.204:443: connect: connection refused
    websocket_test.go:143: timed out waiting for the condition
```
The error cannot happen on my local env so CI has some issue.

Reply:
Opened https://github.com/knative/serving/issues/11387 for this issue.

Reply:
Let's add this issue as a TODO to all of them.